lib/spire/version.rb

```diff
@@ -1,3 +1,3 @@
 module Spire
-  VERSION = "0.7.0"
+  VERSION = "0.7.1"
```

Comment:
@Dujota I would actually call this 0.8.0. We've added new features rather than fix existing, so I would increase the "Minor" aspect rather than the "Maintenance aspect".